src/utils/InputValidators.js

```diff
@@ -141,6 +143,32 @@ export function CidrValidator(cidr) {
   return retValue;
 }
 
+export function AddressesValidator(addresses) {
+  let retValue = {
+    isValid: true,
+    errorMsg: ''
+  };
+
+  // just one IPV4 address
+  if(addresses && addresses.indexOf('-') === -1) {
+    if(IPV4ADDRESS.exec(addresses.trim()) === null) {
+      retValue.isValid = false;
+      retValue.errorMsg = translate('input.validator.addresses.error');
+      return retValue;
+    }
+  }
+
+  if(addresses && addresses.indexOf('-') !== -1) { // just one range
```

Comment:
This really needs some basic logic to cover some simple things like the first address must be less than the second address.

Reply:
Ok, thanks. Will add one more basic logic based on @GarySmith's suggestion. For more complicated network logic validation, we will depend on the configure processor.